ci/do_ci.sh

```diff
@@ -56,14 +56,11 @@ elif [[ "$1" == "cmake.exporter.prometheus.test" ]]; then
 #  apt-get install sudo
 #  apt-get install zlib1g-dev
 #  apt-get -y install libcurl4-openssl-dev
-  cd third_party
-  git clone https://github.com/jupp0r/prometheus-cpp
-  cd prometheus-cpp
-  git checkout v0.9.0
-  git submodule init
-  git submodule update
+  cd third_party/prometheus-cpp
+  git submodule update --recursive --init
+  [[ -d _build ]] && rm -rf ./_build
   mkdir _build && cd _build
-  cmake .. -DBUILD_SHARED_LIBS=ON
+  cmake .. -DBUILD_SHARED_LIBS=ON -DUSE_THIRDPARTY_LIBRARIES=ON
```

Comment:
Use deps from prometheus build tree, to ensure we build all with the matching compiler.